tools/train/embeddings_retrain_preparation.ipynb

```diff
@@ -0,0 +1,1030 @@
+{
```

Comment:
<p>I checked in more details.</p><p> </p><p>The weights we want to reinit are:</p><ul><li>params.lm_head.* (kernel + bias if present)</li><li>params.model.decoder.embed_tokens.*</li><li>params.model.decoder.embed_positions.*</li><li>params.model.decoder.final_ln.*</li><li>params.model.decoder.layernorm_embedding.*</li></ul><p> </p><p>For the encoder I would not reinit because it's on the text only.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/borisdayma/dalle-mini/pull/290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Thanks, I'll update it! I did not reinit the encoder layers but was not sure about `lm_head`.